pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[project]
```

Comment:
It looks like the repository also has a `requirements.txt` file. Only one method should be used to manage dependencies (prefer `pyproject.toml` as it is the newer way to do it).